diff --git a/app.vue b/app.vue
index ba41a70..8952a07 100644
--- a/app.vue
+++ b/app.vue
@@ -71,6 +62,7 @@ const {
 const APP_DEFAULT_LOCAL = 'de'
 
 import fs from 'fs'
+import { debug } from 'console'
 const TREE_DATA_FILE = '/tmp/tree_data.json'
 
 
@@ -134,17 +126,26 @@ if (import.meta.client) {
 router.beforeEach((to, from, next) => {
   const ts = to.name?.toString() || ''
   const fs = from.name?.toString() || ''
-  console.log("router beforeEach: from " + fs + " -> to:" + ts)
+  console.log("router beforeEach: from: " + fs + " -> to: " + ts)
+
+  if (fs === 'index' && ts.indexOf('setlist') > -1) {
+    isShowPageIn.value = true
+  }
+  if (ts === 'index') {
+    isShowPageIn.value = false
+    //updateTheme('dark')
+  }
+  if (ts.indexOf('setview') > -1 || fs.indexOf('setview') > -1) {
+    isNoClip.value = true;
+  } else {
+    isNoClip.value = false;
+  }
+  //if (ts.indexOf('setlist') > -1) {
+    //updateTheme('light');
+  //}
+
   const config = useRuntimeConfig()
   if (!config.public.kioskSetId) {
-    if (ts.indexOf('setview') > -1 || fs.indexOf('setview') > -1) {
-      isNoClip.value = true;
-    } else {
-      isNoClip.value = false;
-    }
-    if (ts.indexOf('setlist') > -1) {
-      updateTheme('light');
-    }
     next()
   }
   else {
@@ -165,7 +166,6 @@ router.beforeEach((to, from, next) => {
       
     }
   }
-  
 })
 
 const isShowPageIn = ref(false)
@@ -186,14 +186,7 @@ watch(() => route.fullPath, () => {
       }, 2000)
     }
   }
-  // dont confuse running animation
-  setTimeout(() => {
-    //console.error(" switch anim mode " + isShowPageIn.value)
-    isShowPageIn.value = !isShowPageIn.value
-  },3000)
-
-  //console.log("APP: changed route: " + JSON.stringify(route.fullPath));
-
+  console.log("APP: changed route: " + JSON.stringify(route.fullPath));
 });
 
 const updateTheme = (defaultTheme:string) => {
@@ -221,6 +214,9 @@ const onkeyupEv = (ev:KeyboardEvent) => {
     updateTheme()
     console.log('switched to high contrast: ' + highContrastState.value)
   }
+  if (ev.altKey && ev.ctrlKey && ev.code == 'KeyI') {
+    router.push('/')
+  }
 }
 const MOBILE_SWITCH_RESOLUTION = 768
 const updateMobileStateByWinWidth = () => {
@@ -240,7 +236,6 @@ const updateMobileStateByWinWidth = () => {
 const highContrastState = useState('isHighContrast')
 
 onMounted(() => {
-  //document.documentElement.setAttribute("data-theme", "light");
   highContrastState.value = 
     window.getComputedStyle(document.body).getPropertyValue('--high-contrast-enabled') == '1000'
   updateTheme('light')
@@ -267,6 +262,13 @@ onBeforeUnmount(() => {
 </script>
 <style>
 
+:root {
+--clip-anim-ms: 800ms;
+--clip-radius: calc(sqrt( pow(100vw) + pow(100vh)));
+--clip-radius: max(100vw,100vh)
+}
+
+
 .page {
   opacity: 1;
 }
@@ -274,12 +276,12 @@ onBeforeUnmount(() => {
 /* clip circle page out ani */
 .page-out.page-enter-active {
   z-index: 5;
-  transition: all 0ms;
+  opacity: 0.99;
+  transition: all var(--clip-anim-ms);
 }
 .page-out.page-leave-active {
-  /*max(100vw, 100vh) */
-  clip-path: circle(calc(sqrt( pow(100vw) + pow(100vh))) at 50vw 50vh) !important;
-  transition: clip-path 800ms ease-out;
+  clip-path: circle(var(--clip-radius) at 50vw 50vh) !important;
+  transition: clip-path var(--clip-anim-ms) ease-out;
   z-index: 10;
 }
 .page-out.page-leave-to {
@@ -292,17 +294,18 @@ onBeforeUnmount(() => {
 /* clip circle page in ani */
 .page-in.page-enter-active {
   clip-path: circle(0vw at 50vw 50vh) !important;
-  transition: clip-path 800ms ease-out;
+  transition: clip-path var(--clip-anim-ms) ease-out;
   z-index: 10;
 }
 .page-in.page-leave-active {
   z-index: 5;
-  transition: all 0ms;
+  transition: all var(--clip-anim-ms);
 }
 .page-in.page-leave-to {
+  opacity: 0.99;
 }
 .page-in.page-enter-to {
-  clip-path: circle(150vw at 50vw 50vh) !important;
+  clip-path: circle(var(--clip-radius) at 50vw 50vh) !important;
 }
 
 /* fade page io ani */
diff --git a/pages/index.vue b/pages/index.vue
index ea5fbfa..2538b64 100644
--- a/pages/index.vue
+++ b/pages/index.vue
@@ -1,6 +1,8 @@
 <template>
+  <div class="intro_page"
+    data-theme="dark"
+    @click.prevent.once="switchPage()">
   <!-- TODO high contrast : at first we can ommit special handling for this page -->
-  <div class="intro_page" data-theme="dark" @click.prevent.once="switchPage()">
   <!-- TODO mobile theme: dynamic fonts font sizes -->
   <!-- TODO animations: individual timings for each intro line -->
     <header>
@@ -62,8 +64,6 @@ const {
 } = treeHelper()
 
 const {
-  font_selected,
-  font_list, 
   selectRandomFont,
   getViewSizedStyle
 } = DynFonts()
@@ -73,23 +73,18 @@ const isKiosk = ref(false)
 const kioskIntroTextLine = ref([] as string[])
 
 const switchPage = () => {
-  const config = useRuntimeConfig()
-  if (config.public.kioskSetId) {
-    isKiosk.value = true
-    const kioskId = config.public.kioskSetId
+  if (isKiosk.value) {
+    const kioskId = useRuntimeConfig().public.kioskSetId
     const path = '/setview/projekt/' 
       + kioskId
       + '/' 
       + kioskId
     router.push(path)
-    kioskIntroTextLine.value = config.public.kioskIntroTextLines as string[]
-  } else if (config.public.kioskForestSetId) {
+  } else if (useRuntimeConfig().public.kioskForestSetId) {
     router.push('/setlist/' + MATCH_PROJECTS)
-    
   } else {
     router.push('/setlist/' + MATCH_DIPLOM)
   }
-  
 }
 
 
@@ -134,7 +129,7 @@ onMounted(() => {
   for (let i = 0; i < 6; i++) {
     lineStyles.value[i] = getFontStyle(i)
   }
-  document.documentElement.setAttribute("data-theme", "dark");  
+  
   // animate in text lines
   setTimeout(() => {
     showContent.value=true
diff --git a/pages/setlist/[settype]/index.vue b/pages/setlist/[settype]/index.vue
index 6a2e66a..6f84d9c 100644
--- a/pages/setlist/[settype]/index.vue
+++ b/pages/setlist/[settype]/index.vue
@@ -1,5 +1,5 @@
 <template>
-  <div class="setlist_page">
+  <div class="setlist_page" data-theme="light">
     <SetListHeader
       :settype="settype"
       :hide-nav-btns="showMenuView || animateSwitch2Set || showFilterView"
@@ -469,7 +469,7 @@ onMounted(() => {
   
   updateSetType();
   isKiosk.value = useRuntimeConfig().public.kioskSetId?.length > 0 || useRuntimeConfig().public.kioskForestSetId?.length > 0
-
+  
   document.documentElement.setAttribute("lang", locale.value );
 
   /* Metadata & SEO */
